.github/workflows/build-preview.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
 
       - name: Install Ruby packages
         if: github.event.action != 'closed'
-        uses: ruby/setup-ruby@v1.172.0
+        uses: ruby/setup-ruby@v1
```

Comment:
This is the fix that was patched in the template a few versions ago, which should fix the ruby error you're getting in the build preview.

The fix is here, yet the build is still failing, because I believe the PR workflow wont actually update until its changes are merged into main. So, it should work on the next PR.